node/api/apistruct/struct.go

```diff
@@ -0,0 +1,26 @@
+package apistruct
```

Comment:
Does this need to be a separate package?

Reply:
It does not -- I had everything broken into packages like this to make it easier to mimic lotus's layout. I have sense created a "lily" lens and moved the LilyNode API and struct into it.